app/src/controllers/testItem/sagas.js

```diff
@@ -96,7 +96,9 @@ function* fetchTestItems() {
         'filter.eq.launch': launchId,
         'filter.eq.parent': !noChildFilter ? parentId : undefined,
         'filter.level.path': sizePath,
-        'filter.under.path': noChildFilter ? parentId : undefined,
+        'filter.under.path': noChildFilter
+          ? itemIds.filter((item) => item !== launchId.toString()).join('.')
```

Comment:
Is it necessary to use `toString` here?

Reply:
`itemIds` is represented as an array of strings, but `launchId` may be a `Number` according to new API

Reply:
Maybe we should convert all ids to number in selectors to reflect API changes?

Reply:
We get `itemIds` from `location` in `state`
Should I convert this items to numbers in appropriate selector?
![image](https://user-images.githubusercontent.com/31861805/48079998-b259b880-e1fd-11e8-9afe-eab91da16540.png)

Reply:
Done